### ISSUE TITLE
Instance templates do not in their current form allow creation of scratch disks

### ISSUE THREAD

**Issue by symbiont-jon-bogaty** [terraform-google-modules/terraform-google-vm#76]
Instance templates do not in their current form allow creation of scratch disks. The missing parameters mean that you end up defaulting to incompatible flags when you try to setup NVME, etc... I've submitted a PR (#75 ) that attempts to alleviate that problem by adding in the missing parameters.

**Comment by homelessnessbo** [comment]
The changes proposed in the PR are already in master, but the problem still persists:

```
Error: Error creating instance template: googleapi: Error 400: Invalid value for field 'resource.properties.disks[1].initializeParams.diskType': 'local-ssd'. Cannot create local SSD as persistent disk., invalid

  on .terraform/modules/store-euwe4.mig_template/terraform-google-vm-1.4.1/modules/instance_template/main.tf line 57, in resource "google_compute_instance_template" "tpl":
  57: resource "google_compute_instance_template" "tpl" {
```

**Comment by evanrich** [comment]
Still an issue

```
Error: 1 error occurred:
	* disks with a disk_type of local-ssd must be SCRATCH disks. disk 1 is a  disk
```
using the following "additional_disks":
```
  additional_disks = [
    {
      disk_size_gb = 375
      disk_type    = "local-ssd"
      type         = "SCRATCH"
      auto_delete  = "true"
      interface    = "NVME"
      boot         = "false"
    },
  ]
```

**Comment by amontalban** [comment]
Should have been fixed with #150.
